In [3]:
x_bar = 3050 #sample mean
n= 25 #sample size
sigma= 5*25 #units
mean=1000+5*600 #theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)

In [1]:
import scipy.stats as stats
import numpy as np
import math

**Stating Hypothesis**

# This is one tailed test.

H0 (Null Hypothesis): The weekly operating cost is equal to or less than the model's cost.

H1 (Alternative Hypothesis): The weekly operating cost is greater than the model's cost.

In [4]:
# Calculate the test statistics by using formula and using given data:

t_stats=(x_bar - mean) / (sigma / math.sqrt(n))
t_stats

-38.0

In [5]:
# Determine the Critical Value using, one tailed test at alpha = 0.05
alpha=0.05
CI=0.95 # confidence interval is (1-alpha)

In [6]:
critical_value=stats.norm.ppf(0.95)
critical_value

1.6448536269514722

**Decision**

In [7]:
if t_stats > critical_value:
    print("Reject the null hypothesis, That means The weekly operating cost is greater than the model's cost")
else:
    print("Fail to reject the null hypothesis,weekly operating cost is equal or less than the model's cost")

Fail to reject the null hypothesis,weekly operating cost is equal or less than the model's cost


Conclusion

There is no significant evidence to support the claim that the weekly operating costs are higher than the model suggests. Therefore, the theoretical cost model still holds based on this sample data.


Since the test statistic (-38.00) is much smaller than the critical value (1.64), this means that the sample data does not provide enough evidence to suggest that the actual weekly operating costs are significantly higher than what the cost model predicts.

HENCE

The data does not support the restaurant owners' claim that the weekly operating costs have increased beyond the predicted levels of the cost model. Therefore, we continue to assume that the current model is accurate for now.